addons/addon-base-raas-ui/packages/base-raas-ui/src/parts/dashboard/Dashboard.js

```diff
@@ -242,13 +273,19 @@ class Dashboard extends React.Component {
   }
 
   renderYesterdayCostPerEnv() {
-    if (_.isEmpty(this.state.envNameToCostInfo)) {
+    if (_.isEmpty(this.state.envIdToCostInfo)) {
       return <ProgressPlaceHolder />;
     }
     const title = "Yesterday's Env Cost";
-    const labels = Object.keys(this.state.envNameToCostInfo);
-    const dataPoints = Object.keys(this.state.envNameToCostInfo).map(envName => {
-      return this.state.envNameToCostInfo[envName].yesterdayCost;
+    const labels = Object.keys(this.state.envIdToCostInfo).map(envId => {
```

Comment:
This block looks the same as line 259 to 265, is there a way to write it only once?

Reply:
Good point, I moved it to a different function and added a unit test for it too